spec/features/idv/steps/in_person_opt_in_ipp_spec.rb

```diff
@@ -0,0 +1,453 @@
+      expect_in_person_step_indicator_current_step(t('step_indicator.flows.idv.secure_account'))
+      expect(page).to have_content(t('titles.idv.personal_key'))
+      deadline = nil
+      freeze_time do
+        acknowledge_and_confirm_personal_key
+        deadline =
+          (Time.zone.now + IdentityConfig.store.in_person_enrollment_validity_in_days.days).
+            in_time_zone(Idv::InPerson::ReadyToVerifyPresenter::USPS_SERVER_TIMEZONE).
+            strftime(t('time.formats.event_date'))
+      end
+
+      # ready to verify page
+      expect_in_person_step_indicator_current_step(
+        t('step_indicator.flows.idv.go_to_the_post_office'),
+      )
+      expect_page_to_have_no_accessibility_violations(page)
+      enrollment_code = JSON.parse(
+        UspsInPersonProofing::Mock::Fixtures.request_enroll_response,
+      )['enrollmentCode']
+      expect(page).to have_css("img[alt='#{APP_NAME}']")
+      expect(page).to have_content(t('in_person_proofing.headings.barcode').tr(' ', ' '))
+      expect(page).to have_content(Idv::InPerson::EnrollmentCodeFormatter.format(enrollment_code))
+      expect(page).to have_content(
+        t(
+          'in_person_proofing.body.barcode.deadline',
+          deadline: deadline,
+        ),
+      )
+      expect(page).to have_content('MILWAUKEE')
+      expect(page).to have_content('Sunday: Closed')
+
+      # signing in again before completing in-person proofing at a post office
+      Capybara.reset_session!
+      sign_in_live_with_2fa(user)
+      visit_idp_from_sp_with_ial2(:oidc)
+      expect(page).to have_current_path(idv_in_person_ready_to_verify_path)
+
+      # confirm that user cannot visit other IdV pages before completing in-person proofing
+      visit idv_agreement_path
+      expect(page).to have_current_path(idv_in_person_ready_to_verify_path)
+      visit idv_ssn_url
+      expect(page).to have_current_path(idv_in_person_ready_to_verify_path)
+      visit idv_verify_info_url
+      expect(page).to have_current_path(idv_in_person_ready_to_verify_path)
+
+      # Confirms that user can visit account page even if not completing in person proofing
+      Capybara.reset_session!
+      sign_in_and_2fa_user(user)
+      expect(page).to have_current_path(account_path)
+    end
+  end
+
+  context 'when ipp_enabled is false and ipp_opt_in_enabled is true' do
+    let(:user) { user_with_2fa }
+    let(:sp) { :oidc }
+
+    before do
+      allow(IdentityConfig.store).to receive(:in_person_proofing_enabled) { false }
+      allow(IdentityConfig.store).to receive(:in_person_proofing_opt_in_enabled) { true }
+    end
+
+    it 'skips how to verify and continues along the normal path' do
+      sign_in_and_2fa_user(user)
+      visit_idp_from_sp_with_ial2(:oidc)
+      complete_welcome_step
+      complete_agreement_step
+      expect(page).to have_current_path(idv_hybrid_handoff_url)
+      complete_hybrid_handoff_step
+      complete_document_capture_step
+      complete_ssn_step
```

Comment:
If we move from doc capture to ssn we are not in idv. In idv flow- I feel like we should hit a page that says try again because the user cannot move forward and that is what we should be checking for.  This situation should not happen (no one should be setting this combo- but feel we should test for it because it can happen).  Thoughts? Maybe we talk more about this before you begin work?

Reply:
Let's revisit via Slack messages and huddle if needed.

Reply:
After more talking, more think, and more talking...  I think we are fine as is.  The purpose of this test is to show we skip the How to Verify page, and it does just that! Eventually we end up in remote flow (which is expected) with the flags. I think what I was suggesting is out of scope for this ticket as it shows the work flow past How to Verify page. 

No changes needed.